spec/lib/data_food_consortium/connector/connector_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require Rails.root.join('lib/data_food_consortium/connector/connector')
+
+describe DataFoodConsortium::Connector::Connector, vcr: true do
+  subject(:connector) { described_class.instance }
+  let(:product) do
+    DataFoodConsortium::Connector::SuppliedProduct.new(
+      "https://example.net/tomato",
+      name: "Tomato",
+      description: "Awesome tomato"
+    )
+  end
+
+  it "exports" do
+    json = connector.export(product)
+    expect(json).to match '"dfc-b:name":"Tomato"'
+  end
+
+  it "imports" do
+    json = connector.export(product)
```

Comment:
> wouldn't it be best practice to predefine the DFC json (in a fixture file)?

Hm, yes, there are two different tests:

1. Does the import function reverse the export function?
2. Does the import function deal with existing documents?

I'm only testing the first one at the moment and it would be good to test the second one with fixtures to know if we break integrations.